src/OrbitGl/CaptureViewElementTest.cpp

```diff
@@ -172,6 +172,16 @@ TEST(CaptureViewElement, IsMouseOver) {
   EXPECT_FALSE(container_elem.IsMouseOver());
   EXPECT_FALSE(child0->IsMouseOver());
   EXPECT_FALSE(child1->IsMouseOver());
+
+  // Moving as part of left-click dragging shouldn't modify mouse_over. This is because highlighting
+  // of some elements rely on that and we only want to highlight them when we move the mouse without
+  // any modifier.
```

Comment:
NIT: Mouse moving when left-clicking + dragging should not update `IsMouseOver()` flag. This is some elements are highlighted when `IsMouseOver()` returns true. But we want to avoid highlighting them when we move the mouse without any modifier. 

Such that you comments keep consistent with previous one.